### PR TITLE
IH Badge Additions and Changes [FUCKING Finally Works]

### PR DESCRIPTION
**Pull request by thevandie**
## About The Pull Request

Adds multiple IH badges as well as replacing a few old ones. The holobadges have 2 frames, so they flicker, btw.

## Why It's Good For The Game

Giving certain roles like the Sergeant, Specialist, and Inspector their own unique badges to differentiate them from the operatives makes them more unique and fun.

## Testing

I have tested everything ingame. They work. I tested everything on the redo, it works. Everything works ingame. I am going to sleep now, good night.
badges
badgesingame
image

## Changelog
:cl:
add: Sergeant, Medspec, and Inspector badges
imageadd: New sprites for the Gunny badge, Marshal badge, and Commander badge as well as Medspec Sarge and Inspector sprites for the new items.
tweak: Slight flavor changes for the badges to make them fit better.
/:cl:

### CODE REVIEW

File: code/modules/clothing/accessories/badges.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 /obj/item/clothing/accessory/badge
 	name = "Inspector's badge"
```
Comment (Pink-Chink) [inline code/modules/clothing/accessories/badges.dm:8]: Should give this a different name since "Inspector's badge" is already used.

Comment (thevandie) [reply code/modules/clothing/accessories/badges.dm:8]: Better idea, what if I just made the old inspector's badge distinctly old?

Comment (thevandie) [reply code/modules/clothing/accessories/badges.dm]: Now they're distinct, boom

File: code/modules/clothing/accessories/badges.dm [hunk 2]
```diff
@@ -5,8 +5,8 @@
 */
 
 /obj/item/clothing/accessory/badge
-	name = "Inspector's badge"
-	desc = "A Ironhammer Security badge, made from gold and set on false leather."
+	name = "Faded Inspector's badge"
```
Comment (Pink-Chink) [inline code/modules/clothing/accessories/badges.dm:8]:
```suggestion
	name = "faded Inspector's badge"
```

**Review by Pink-Chink — APPROVED** [review]
The inspector is faded.

**Comment by thevandie** [comment]
IT WORKS! IT ALL FUCKING WORKS! I CAN GO THE FUCK TO SLEEP FINALLY

**Pull request merged**